clients/lsp-yaml.el

```diff
@@ -161,7 +161,7 @@ Limited for performance reasons."
                                      `(,(or (executable-find (cl-first lsp-yaml-server-command))
                                             (lsp-package-path 'yaml-language-server))
                                        ,@(cl-rest lsp-yaml-server-command))))
-                  :major-modes '(yaml-mode k8s-mode)
+                  :major-modes '(yaml-mode k8s-mode gitlab-ci-mode)
```

Comment:
What's the file extension?
Perhaps we should go with something like `:activation-fn (lsp-activate-on "yaml")` and define the regex for file name (maybe `.*\\.(yaml|yml)`) instead

Reply:
The file extension is ".yml".

So I pushed a new commit with a more generic solution.